Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/core/VaccinationCertificate.kt

```diff
@@ -1,20 +1,24 @@
 package de.rki.coronawarnapp.vaccination.core
 
 import de.rki.coronawarnapp.ui.Country
-import org.joda.time.Instant
 import org.joda.time.LocalDate
 
-data class VaccinationCertificate(
-    val firstName: String,
-    val lastName: String,
-    val dateOfBirth: LocalDate,
-    val vaccinatedAt: Instant,
-    val vaccineName: String,
-    val vaccineManufacturer: String,
-    val chargeId: String,
-    val certificateIssuer: String,
-    val certificateCountry: Country,
-    val certificateId: String,
-) {
-    val identifier: VaccinatedPersonIdentifier get() = ""
+interface VaccinationCertificate {
+    val firstName: String?
+    val lastName: String
```

Comment:
How about  aligning naming here with Tech specs ? `givenName and familyName `

Reply:
I always find this very confusing and I'm more used to first/last name. If the other devs do not share this opinion, then I'm fine to refactor this in a follow up PR.